code/src/components/style/All.css

```diff
@@ -0,0 +1,44 @@
+body {
+  background: no-repeat center center fixed;
+  background-color: #febba7;
+  background-image: linear-gradient(19deg, #febba7 0%, #d7ebef 100%);
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+.all {
+  /* width: 100%; */
```

Comment:
Maybe here you can add a min and max width to not make it go all the way on desktop (and not be super thin when the list is empty):
```
min-width: 250px;
max-width: 600px;
```